### DISCUSSION TITLE
Can this code base support more classes then coco dataset like 500-600 classes

### DISCUSSION THREAD

**Discussion by ghost** [ultralytics/yolov5#2975] [General]
Hi, I dont have much experience in this field and i want to train a model with 500 classes(different objects). Just wondering if this would be capable of doing it effectively the way it does for current 80 classes. And would it have problems when have more classes in future like 1000s of classes?

**Comment by glenn-jocher** [comment]
@wuclid26vj there is no constraint on class count. Results will vary based on a variety of factors.
